examples/with-tailwind/packages/ui/package.json

```diff
@@ -1,34 +1,33 @@
 {
-  "name": "ui",
+  "name": "@repo/ui",
   "version": "0.0.0",
   "sideEffects": [
     "**/*.css"
   ],
-  "types": "./dist/index.d.ts",
   "exports": {
-    ".": "./dist",
-    "./styles.css": "./dist/index.css"
+    "./styles.css": "./dist/index.css",
+    "./card": "./src/card.tsx"
   },
   "license": "MIT",
   "scripts": {
-    "build": "tsup",
+    "build": "tailwindcss -i ./src/styles.css -o dist/index.css",
     "lint": "eslint src/",
-    "dev": "tsup --watch",
-    "check-types": "tsc --noEmit"
+    "dev": "tailwindcss -i ./src/styles.css -o ./dist/index.css --watch",
```

Comment:
This task is working correctly when ran outside of `turbo` but exits early when ran with `turbo`.

Reply:
Update: This task works on `canary`.